src/daemon/dlt-daemon.c

```diff
@@ -1636,6 +1636,13 @@ int main(int argc, char *argv[])
     dlt_gateway_deinit(&daemon_local.pGateway, daemon_local.flags.vflag);
 
     dlt_daemon_free(&daemon, daemon_local.flags.vflag);
+#ifdef DLT_TRACE_LOAD_CTRL_ENABLE
```

Comment:
Those lines of code should be warped in 1 function like "dlt_traceload_free"

Reply:
Will do, once I'm back in office in December :)